tsz-compress/config.toml

```diff
@@ -0,0 +1,3 @@
+# Determines the compression algorithm (delta or delta-delta) based on relative size differences.
+# A higher value increases the likelihood of using both algorithms, while a lower value leads to an early decision.
+COMPRESSION_SIZE_FACTOR = 5
```

Comment:
Normally this type of parameter is given when creating the Compressor instance. Here is an example where polars takes a runtime configurable compression level that is algorithm dependent.
https://github.com/pola-rs/polars/blob/f3ee4d08842f8ad8f18bab69378c8b322f677996/crates/polars-io/src/parquet/write.rs#L55